adafruit_veml6070.py

```diff
@@ -145,18 +145,18 @@ def __init__(self, i2c_bus, _veml6070_it="VEML6070_1_T", ack=False):
 
 
     @property
-    def read(self):
+    def raw_uv(self):
```

Comment:
How about `uv_raw`? That'd be more inline with the VEML6075 that has `uva` and `uvb` properties.